remoteworkitem/remoteworkitem.go

```diff
@@ -22,6 +22,12 @@ var WorkItemKeyMaps = map[string]WorkItemMap{
 		AttributeExpression("state"): SystemStatus,
 		AttributeExpression("id"):    SystemRemoteItemId,
 	},
+	ProviderJira: WorkItemMap{
+		AttributeExpression("fields.summary"):     SystemTitle,
+		AttributeExpression("fields.description"): SystemDescription,
+		AttributeExpression("fields.status.name"): SystemStatus,
```

Comment:
Verify the 'target' fields match the current WIT's; https://github.com/almighty/almighty-core/blob/master/migration/migration.go#L52